### ISSUE TITLE
Pre-Train EAT on own dataset

### ISSUE THREAD

**Issue by aalbayrak394** [cwx-worst-one/EAT#7]
Hi, is there any tutorial on how to pre-train EAT on own data? I'd like to pre-train it on a LightningDataModule, but I have no clue if it's compatible with all this fairseq stuff. 
I would be very grateful for an answer and/ or little instruction :).

**Comment by cwx-worst-one** [comment]
Sure! If you need to pre-train on your own dataset, you can simply modify the parameters in `EAT/scripts/pretraining_AS2M.sh`, specifically `task.data=your_data_manifest_folder`, and run this script. To create a manifest file from your data, you can refer to our provided [example](https://drive.usercontent.google.com/download?id=1LH2C0q3d4zndoR3-oGkVdYYqDCIdxIsm&export=download&authuser=0) and its README.

By the way, during the training process, you might need to adjust the [global normalization parameters](https://github.com/cwx-worst-one/EAT/blob/65ac1a24555ec036ca507d7261358027ed6219ad/data/raw_audio_dataset.py#L415C30-L415C31), which are the mean and std of the Mel spectrograms for all the audio used for normalization. However, this might not have a significant impact on the pre-training performance of the model.
